clientv3/balancer.go

```diff
@@ -47,6 +47,15 @@ type simpleBalancer struct {
 	// upc closes when upEps transitions from empty to non-zero or the balancer closes.
 	upc chan struct{}
 
+	// downc closes when grpc calls down() on pinAddr or the balancer closes.
+	downc chan struct{}
+
+	// stopc closes when sendLoop() is done.
+	stopc chan struct{}
```

Comment:
`stopc`'s are used to signal to goroutines to shutdown, not to signal that all goroutines have been torn down